lib/coverage.js

```diff
@@ -412,6 +412,21 @@ internals.addSourceMapsInformation = function (ret, num) {
         source.originalFilename = ret.filename;
         source.originalLine = num;
     }
+
+    if (source.chunks) {
+        source.chunks.forEach((chunk) => {
+            // Also add source map information on chunks
+            originalPosition = SourceMapSupport.mapSourcePosition({
+                source: ret.filename,
+                line: num,
+                column: chunk.column
+            });
+            chunk.originalFilename = originalPosition.source.replace(Path.join(process.cwd(), '/').replace(/\\/g, '/'), '');
```

Comment:
I've copy-pasted this from [line 404](https://github.com/hapijs/lab/pull/648/files/b95f9616e312574bb1071e8f1337a3ef2a8d0f11#diff-1c3512bd280f1fe9c07481535be0d978R404), but I'll give a try, because using basename is definitely more understandable !